rust/connlib/snownet/src/node.rs

```diff
@@ -733,13 +733,12 @@ where
                 self.private_key.clone(),
                 remote,
                 Some(key),
-                Some(25), // 25 is the default of the kernel implementation.
+                self.mode.is_client().then_some(25), // 25 is the default of the kernel implementation.
```

Comment:
Is this the persistent keepalive? Just FYI, by default it's off for the kernel implementation (to be "as silent as possible").